lib/tjbot.js

```diff
@@ -199,8 +199,23 @@ TJBot.prototype._setupCamera = function() {
         encoding: 'jpg',
         outputDir: './',
         verticalFlip: this.configuration.see.camera.verticalFlip,
-        horizontalFlip: this.configuration.see.camera.horizontalFlip
+        horizontalFlip: this.configuration.see.camera.horizontalFlip,
+        time: 1
```

Comment:
i believe it's `time`, as per node-raspistill's options:
https://github.com/dlukanin/node-raspistill/blob/master/src/lib/camera/abstract.ts

```
/**
     * Map "option and option value -> raspistil exec arg"
     * @type {any}
     */
    protected optionsMap: {[key: string]: string} = {
        verticalFlip: '-vf',
        horizontalFlip: '-hf',
        noPreview: '-n',
        encoding: '-e',
        width: '-w',
        height: '-h',
        timelapse: '-tl',
        time: '-t'
    };
```